VTUL/iawa
# LIBTD-1793: Rails task to run batch import by reading from input file…

… about the CSV file and file structures.

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1793) (:star:)

# What does this Pull Request do? (:star:)
This PR creates a Rails task to perform batch importing metadata and building parent-child relationships between collections and items.

# What's the changes? (:star:)

* Add a batch import task
* Batch create metadata for items
* Batch build collection/subcollection/item parent-child relationships

# How should this be tested?

* Run Installscripts for iawa in production mode
* Set iiif_manifest_base_url: "https://img.cloud.lib.vt.edu/iawa/" in ansible/site_secrets.yml 
* Run ./bin/rake iawa:batch_import RAILS_ENV=production
* Should see from the application: all the related collections/items are created with metadata information and in each item display page, image files are displayed in Mirador viewer

# Additional Notes:

* What branch to be used for testing this PR? Branch LIBTD-1793

# Interested parties
Tag (@yinlinchen @pmather ) interested parties

(:star:) Required fields
